operator/controllers/config_mgt.go

```diff
@@ -39,11 +39,11 @@ func prepareDeployInfos(ctx context.Context, manifestObj *v1alpha1.Manifest, def
 	}
 	installConfigObj, ok := decodedConfig.(map[string]interface{})
 	if !ok {
-		return nil, fmt.Errorf(configReadError)
+		return nil, fmt.Errorf("can not decodedConfig")
 	}
 	configs, ok := installConfigObj["configs"].([]interface{})
 	if !ok {
-		return nil, fmt.Errorf(configReadError)
+		return nil, fmt.Errorf("can not parse configs field")
```

Comment:
return nil, fmt.Errorf("cannot parse 'configs' inside default configuration for Manifest %s", namespacedName)